### DISCUSSION TITLE
router cannot be rebooted from CLI or OpenWRT

### DISCUSSION THREAD

**Discussion by alhaddar** [AdguardTeam/AdGuardHome#2745] [Q&A]
Hi
router cannot be rebooted from CLI or OpenWRT  and I removed / opt / AdGuardHome / AdGuardHome
And everything is normal, but I cannot enter the program link
192.168.1.1: 8080, and I can only access the Internet after I return the line / opt / AdGuardHome / AdGuardHome in its previous location.
Please help solve this problem

**Comment by ameshkov** [comment]
Well, I am not sure how exactly you installed AdGuard Home.

Maybe you also need to reconfigure router's dnsmasq to not use AGH anymore.
